data/assets/scene/solarsystem/missions/euclid/model.asset

```diff
@@ -0,0 +1,73 @@
+local transform = asset.require("scene/solarsystem/planets/earth/transforms")
+local sun = asset.require("scene/solarsystem/sun/sun")
+local spice = asset.require("./spice")
```

Comment:
The other Pull Request that I have changes the names of all `kernel` files to `spice` since with that change they don't only provide the kernels but also the SPICE names of the things they load